src/Daterangepicker.php

```diff
@@ -20,9 +20,13 @@ public function __construct(
         private string $default = Helper::TODAY,
         private string $orderByColumn = 'id',
         private string $orderByDir = 'asc',
+        public $name = '',
     ) {
         //Often date range components use as default date the past dates
         $this->maxDate = Carbon::today();
+        if ($name) {
+            $this->name = $name;
+        }
```

Comment:
This condition can be removed, `public $name = '',` going to uses the value that is passed in the constructor, if not going to be empty.